group_vars/all

```diff
@@ -233,4 +233,25 @@ skip_fwrkr_tls_certs_generatation: false
 # - (functions worker) server certificate expiry days
 fwrkr_cert_expire_days: 730
 # - (functions worker) server private key expiry days
-fwrkr_key_password: "MyFwrkrSrvCertPass"
\ No newline at end of file
+fwrkr_key_password: "MyFwrkrSrvCertPass"
+
+##
+# Pulsar AdminConsole server: signed certificate related settings
+#
+# - (Admin Console) server certificate expiry days
+anco_cert_expire_days: 730
+# - (Admin Console) server private key expiry days
+anco_key_password: "MyAncoSrvCertPass"
+
+#
+# How to deploy Admin Console
+# - shared: deploy Admin Console as part of brokers
+# - dedicated: deploy Admin Console on dedicated instances
+#
+#
+deploy_adminconsole: "dedicated"
+
+# - Whether or not to enable Admin Console In-Transit TLS encryption
+enable_anco_tls: true
```

Comment:
I would recommend using "enable_anco_https" as the variable name to make it clear that it is for HTTPS access.

Reply:
Change it.